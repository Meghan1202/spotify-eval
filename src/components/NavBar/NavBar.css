.Nav-Bar-Container {
    display: flex;
    background-color: #38497E;
    color: white;
    height: 2rem;
    justify-content: flex-start;
    padding: 0.5rem;
}