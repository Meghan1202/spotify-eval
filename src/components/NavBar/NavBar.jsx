import React from 'react';
import './NavBar.css';

const NavBar = () => (
  <div className="Nav-Bar-Container">
    My &nbsp;
    <strong> Record </strong>
    &nbsp; Shelf
  </div>
);

export default NavBar;
