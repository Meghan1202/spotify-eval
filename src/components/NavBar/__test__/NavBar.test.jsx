import React from 'react';
import { render } from '@testing-library/react';
import NavBar from '../NavBar';

describe(NavBar.name, () => {
  it('should match the snapshot', () => {
    const { container } = render(<NavBar />);
    expect(container).toMatchSnapshot();
  });
});
