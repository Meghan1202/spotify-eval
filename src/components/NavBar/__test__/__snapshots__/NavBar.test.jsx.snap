// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should match the snapshot 1`] = `
<div>
  <div
    class="Nav-Bar-Container"
  >
    My  
    <strong>
       Record 
    </strong>
      Shelf
  </div>
</div>
`;
